File: 08-1-3-Strategy/DecoyDuck.cs
namespace Strategy
{
    using System;

    public sealed class DecoyDuck : Duck
    {
        public override void Quack() => Console.WriteLine("Robotic quack quack!");

        public override void Display() => Console.WriteLine("Looks like mallard duck, but made from metal.");

        public override void Fly() => Console.WriteLine("Can not fly, not real.");
    }
}

File: 10-DesignPatterns-Java/Starter/Strategy/src/DecoyDuck.java
public final class DecoyDuck extends Duck {
    @Override
    public void quack() {
        System.out.println("Robotic quack quack!");
    }

    @Override
    public void display() {
        System.out.println("Looks like mallard duck, but made from metal.");
    }

    @Override
    public  void fly() {
        System.out.println("Can not fly, not real.");
    }
}